mod/wet4/views/default/object/groupforumtopic.php

```diff
@@ -26,9 +26,13 @@
 	return;
 }
 
-
-	$excerpt = elgg_get_excerpt(gc_explode_translation($topic->description, $lang));
-	$description = gc_explode_translation($topic->description, $lang);
+if($topic->description3){
```

Comment:
* all description 3s should have been migrated to the json format in the regular description, were there some that weren't? 
* $topic->description should be passed through gc_explode_translation() or you'll run into issues displaying new bilingual content